EIPS/eip-7092.md

```diff
@@ -0,0 +1,342 @@
+author: Samuel Gwlanold Edoumou (@Edoumou)
+discussions-to: https://ethereum-magicians.org/t/financial-bonds/14461
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-28
+---
+
+## Abstract
+
+The proposed standard allows for the implementation of basic functionality for fixed income financial bonds with smart contracts.
+Principal bonds characteristics such as the bond isin, the issue volume, the issue date, the maturity date, the coupon rate,
+the coupon frequency, the principal, or the day count basis are defined to allow issuing bonds in the primary market (origination),
+and different transfer functions allow to buy or sell bonds in the secondary market. The standard also providses a functionality to
+allow bonds to be approved by owners in order to be spent by third party.
+
+## Motivation
+
+Fixed income instruments are one of the asset classes that is widely used by corporations and other entities to raise funds. Bonds
+are considered more secured than equity since the issuer is supposed to repay the principal at maturity in addition to coupons
+that are paid to investsors.
+
+This standard interface allows fixed income instruments to be represented as on-chain tokens, so as they can be managed through wallets,
+and be used by applications like decentrailized exchanges.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+**Every contract compliant with the [ERC-7092](./eip-7092.md) MUST implement the following interface**
+
+```solidity
+pragma solidity ^0.8.0;
+
+/**
+* @title ERC-7092 Financial Bonds tandard
+*/
+interface IERC7092 {
+    /**
+    *  @notice Returns the bond isin
+    */
+    function isin() external view returns(string memory);
+
+    /**
+    * @notice Returs the bond name
+    */
+    function name() external view returns(string memory);
+
+    /**
+    * @notice Returns the bond symbol
+    */
+    function symbol() external view returns(string memory);
+
+    /**
+    * @notice Returns the numbr of decimals the bond uses - e?g `10`, means to divide the token amount by `10000000000`
+    *
+    * OPTIONAL
+    */
+    function decimals() external view returns(uint8);
+
+    /**
+    * @notice Returns the bond currency. This is the contract address of the token used to pay and return the bond principal
+    */
+    function currency() external view returns(address);
+
+    /**
+    * @notice Returns the copoun currency. This is the contract address of the token used to pay coupons. It can be same as the the one used for the principal
+    */
+    function currencyOfCoupon() external view returns(address);
+
+    /**
+    * @notice Returns the bond denominiation. This is the minimum amount in which the Bonds may be issued. It must be expressend in unnit of the principal currency
```

Comment:
typo: instead of  "unnit" it should be "uint"

Reply:
Yes @fosgate29 it should.

Thx